### ISSUE TITLE
Add a tutorial elecricity and sector-coupled

### ISSUE THREAD

**Issue by davide-f** [pz-max/pypsa-africa#1107]
## Describe the feature you'd like to see

Similar to PyPSA-Eur, we should have an elec-only and sector-coupled tutorial in the documentation to give a head start to users on how to use elec-only and sector-coupled modelling

**Comment by GbotemiB** [comment]
Hi @davide-f, I am very interested in this.

**Comment by ekatef** [comment]
Hey @davide-f, hey @GbotemiB! A great idea and happy to support!

**Comment by GbotemiB** [comment]
Hi @davide-f @ekatef, 

Regarding the tutorial, I think having a dedicated folder for the tutorial in the config folder would be best. We can have the config file for the electricity and the sector-coupled in the folder. 

Another suggestion will be to move the default config to the config folder. We can have all the configs there.

**Comment by GbotemiB** [comment]
In addition, the current tutorial config doesn't allow the option to have scenario naming, which is very important to keep the files detached from each other.

**Comment by ekatef** [comment]
@GbotemiB agree that the configuration files definitely requires some re-structuring. 

Not sure it's a good idea to mix this task with creating the cross-sectoral tutorial, but it may be worth to keep in mind that some further improvement will be needed. Agree that `configs` folder may be a good place to keep the configuration files. Could it be a good idea to create there two sub-folder to keep configs for the tutorial and full-scale runs, respectively?

**Comment by GbotemiB** [comment]
> @GbotemiB agree that the configuration files definitely requires some re-structuring.
> 
> Not sure it's a good idea to mix this task with creating the cross-sectoral tutorial, but it may be worth to keep in mind that some further improvement will be needed. Agree that `configs` folder may be a good place to keep the configuration files. Could it be a good idea to create there two sub-folder to keep configs for the tutorial and full-scale runs, respectively?

Maybe not separate folders, we could just have a tutorial sub-folder and leave the rest as it is.

I also agree to not mix this with the cross-sectoral tutorial, but the tutorial config for the cross-sectoral tutorial will be needed.

**Comment by ekatef** [comment]
> > @GbotemiB agree that the configuration files definitely requires some re-structuring.
> > Not sure it's a good idea to mix this task with creating the cross-sectoral tutorial, but it may be worth to keep in mind that some further improvement will be needed. Agree that `configs` folder may be a good place to keep the configuration files. Could it be a good idea to create there two sub-folder to keep configs for the tutorial and full-scale runs, respectively?
> 
> Maybe not separate folders, we could just have a tutorial sub-folder and leave the rest as it is.
> 
> I also agree to not mix this with the cross-sectoral tutorial, but the tutorial config for the cross-sectoral tutorial will be needed.

Agree, creating a sub-folder for tutorial configs sounds like a good idea and is linked with the task to create a cross-sectoral tutorial. I'd go ahead with that, unless @davide-f @hazemakhalek have any concerns on such design
